modules/swagger-parser-v3/src/test/resources/path-parameter-validation.yaml

```diff
@@ -0,0 +1,25 @@
+openapi: 3.0.0
+info:
+  title: Sample APIs
+  version: 1.0.0
+  description:
+    Sample API
+
+components:
+  parameters:
+    accountId:
+      name: accountId
+      schema:
+        type: string
+      in: path
```

Comment:
added https://github.com/swagger-api/swagger-parser/pull/977/files#diff-e3d6dbd47e8546ab4b5f4fa5fd87146aR1654

Reply:
Great, thanks! 
But now this test doesn't check that case and it doesn't fail. 
Will there be a separate ticket to write an implementation of this test? 
It can be simply forgotten after some time either, I guess.

Reply:
https://github.com/swagger-api/swagger-parser/issues/982
issued!